<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cred</class>
 <widget class="QWidget" name="Cred">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>915</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>915</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="pushButton_calculator">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>340</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_loan">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
 background-color: white;
border: 1px solid black;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>Loan</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_period">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
 background-color: white;
border: 1px solid black;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>Period (in months)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>170</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
 background-color: white;
border: 1px solid black;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>Rate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_period">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>100</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>600</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_loan">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>30</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_rate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Annuity">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>240</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Diff">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>270</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>240</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
 background-color: white;
border: 1px solid black;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>300</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_monthly_payment">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
 background-color: white;
border: 1px solid black;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>Monthly payment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_overpayment">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
 background-color: white;
border: 1px solid black;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_total_payment">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
 background-color: white;
border: 1px solid black;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>Total payout</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly_payment">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>35</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>105</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>175</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
